code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -27,9 +27,9 @@
 
 /obj/item/reagent_containers/pill/get_w_volume()
```

Comment:
so why aren't we just using a linear scaling from a minimum of 1 to a maximum of [x] at [y] volume? the volumetric storage system fully supports decimals, you know.

Reply:
well, I actually didn't know, so thanks for telling. I'll make it scale linearly for sure later.